synchronised_migration.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_dependency 'redlock', '~> 0.2'
```

Comment:
While you're here, it might be nice to drop the version dependencies in this gemspec unless they're _really_ needed